data/kubevirt-ipam-controller/003-passtbindingcni.yaml

```diff
@@ -21,8 +21,14 @@ spec:
         name: passt-binding-cni
         tier: node
         app: passt-binding-cni
+{{ if .IsOpenshift }}
       annotations:
+        openshift.io/required-scc: "passt-binding-cni"
```

Comment:
do we need to create the scc?

Reply:
yes.

Reply:
if so, the PR isnt final yet iiuc

Reply:
?

Reply:
meant that the required SCC might need to be added unless they exists already in the manifests
i see for example this one exists, if all the other already exists / system created we are good
can you please spin it on CRC / kcli to see we are all set please?

```
kind: SecurityContextConstraints
metadata:
  name: passt-binding-cni
```

Reply:
@oshoval it's working:

```sh
oc version
Client Version: 4.17.7
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: 4.17.7
Kubernetes Version: v1.30.6

cat <<EOF | oc apply -f -              
apiVersion: networkaddonsoperator.network.kubevirt.io/v1
kind: NetworkAddonsConfig
metadata:
  name: cluster
spec:
  multus: {}
  linuxBridge: {}
  kubeMacPool: {}
  ovs: {}
  kubeSecondaryDNS: {}
  kubevirtIpamController: {}
  imagePullPolicy: Always
EOF
networkaddonsconfig.networkaddonsoperator.network.kubevirt.io/cluster configured

oc get pods -ncluster-network-addons -w
NAME                                                  READY   STATUS    RESTARTS   AGE
bridge-marker-44qm4                                   1/1     Running   0          65s
cluster-network-addons-operator-6c96bbb9d9-7dcvr      2/2     Running   0          10m
kube-cni-linux-bridge-plugin-zh6bf                    1/1     Running   0          66s
kubemacpool-cert-manager-65d6b697fb-dqjgv             1/1     Running   0          65s
kubemacpool-mac-controller-manager-5db75d6c4d-hcrvf   2/2     Running   0          65s
kubevirt-ipam-controller-manager-7f588c55df-w9jpk     1/1     Running   0          64s
ovs-cni-amd64-d4tsm                                   1/1     Running   0          65s
passt-binding-cni-wh29m                               1/1     Running   0          64s
secondary-dns-6c89d7864d-rvr94                        2/2     Running   0          64s

oc get pods -oyaml -ncluster-network-addons | grep required-scc
      openshift.io/required-scc: bridge-marker
      openshift.io/required-scc: linux-bridge
      openshift.io/required-scc: restricted-v2
      openshift.io/required-scc: restricted-v2
      openshift.io/required-scc: restricted-v2
      openshift.io/required-scc: passt-binding-cni
```